### ISSUE TITLE
Special Abilities

### ISSUE THREAD

**Issue by Ne0Strix** [Ne0Strix/open-realms#120]

**Comment by Ne0Strix** [comment]
I'll start working on the abilities that do not depend on other features. I aim for the following order of implementation:
* draw 1 card
* target opponent discards a card

and then depending on the status of the dependencies the remaining three.

**Comment by Ne0Strix** [comment]
draw 1 card is done, realised that I'm also depending on #121 for the other one; on hold for now

**Comment by Ne0Strix** [comment]
now working on damage per guard as this can be done without any dependencies

**Comment by Ne0Strix** [comment]
As my dependency #121 is not done yet I decided to be *agile* and implement a different set of abilities that does not depend on that. Given the scope of the now implemented abilities I'd consider this branch a complete "feature" as understood in the seminar-requirements.
 
**Note regarding value-pooling:** When champions are played, they only count to the total of champions *after* their first activation when they're played.
